Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/BearerTokenIdentityResolvers/DefaultBearerTokenIdentityResolverChain.swift

```diff
@@ -7,6 +7,7 @@
 
 import protocol SmithyIdentity.BearerTokenIdentityResolver
 import struct SmithyIdentity.BearerTokenIdentity
+@_spi(ClientConfigDefaultIdentityResolver) import protocol SmithyIdentityAPI.ClientConfigDefaultIdentityResolver
```

Comment:
Below, added `ClientConfigDefaultIdentityResolver` conformance to this type.

When it's created by passing in a chain, it's not a client config default.
When it's created with no params, it's a client config default.  (This initializer is package-scoped so customers can't use it.)